server/tests/api/single-pod.js

```diff
@@ -187,14 +187,19 @@ describe('Test a single pod', function () {
   })
 
   it('Should have the views updated', function (done) {
-    videosUtils.getVideo(server.url, videoId, function (err, res) {
-      if (err) throw err
+    this.timeout(5000)
 
-      const video = res.body
-      expect(video.views).to.equal(1)
+    setTimeout(videosUtils.getVideo,
```

Comment:
The failed on my laptop (video.views was equal to 0) so I add this timeout. 1000ms may be enough tough.